plugins/Marketplace/tests/UI/Marketplace_spec.js

```diff
@@ -186,6 +186,15 @@ describe("Marketplace", function () {
             await captureWithPluginDetails('paid_plugin_details_valid_license_' + mode + '_installed');
         });
 
+        it('should show an add to cart button with user selector', async function() {
+            setEnvironment(mode, noLicense);
+
+            var isFree = false;
+            await loadPluginDetailPage('Paid Plugin 1', isFree);
+
+            await captureWithPluginDetails('paid_plugins_no_license_add_to_cart_' + mode);
```

Comment:
```suggestion
            await captureWithPluginDetails('paid_plugin_details_add_to_cart_' + mode);
```

The `paid_plugins` set of screenshot is used for the plugin list, the details modal screenshots are all prefixed with `paid_plugin_details`.

Reply:
I found reading all of them quite confusing, hence the mistake.

Not for now but maybe we should change them to names like you kind of suggested, and split into two files.

`marketplace plugin list tests`
`marketplace plugin modal tests`

I've made your suggested change.